data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -1,28 +1,43 @@
 package io.gnosis.data.repositories
 
+import io.gnosis.data.backend.GatewayApi
 import io.gnosis.data.backend.TransactionServiceApi
-import io.gnosis.data.backend.dto.ServiceTokenInfo
-import io.gnosis.data.backend.dto.tokenAsErc20Token
-import io.gnosis.data.db.daos.Erc20TokenDao
-import io.gnosis.data.models.Balance
-import io.gnosis.data.models.Collectible
-import io.gnosis.data.models.Erc20Token
+import io.gnosis.data.models.*
 import pm.gnosis.crypto.utils.asEthereumAddressChecksumString
 import pm.gnosis.model.Solidity
 import java.math.BigInteger
 
 class TokenRepository(
-    private val erc20TokenDao: Erc20TokenDao,
-    private val transactionServiceApi: TransactionServiceApi
+    private val transactionServiceApi: TransactionServiceApi,
```

Comment:
Seems to me that this dependency is no longer used here.

Reply:
it used for collectibles

Reply:
Actually we do have an endpoint for collectibles in the gateway: `/v1/safes/<safe_address>/collectibles` it just forwards the request to the backend and caches the result. Just letting you know in case you want to implement it. The data models are the same so you would just have to add the method to the `GatewayApi` interface only, AFAICT.

Reply:
not in the scope of this pr